### PR TITLE
[all]: fix spotless linter and enable linting of imports

### PR DESCRIPTION
**Pull request by Jaguar0625**
problem: spotless is not running
    solution: update spotless configuation

     problem: import linting is not enabled
    solution: configure spotless to order imports and remove unused imports

### CODE REVIEW

**Review by Wayonb — APPROVED** [review]
👍🏾

File: nis/src/it/java/org/nem/nis/BlockScorerITCase.java [hunk 1]
```diff
@@ -10,20 +15,14 @@
...
 public class BlockScorerITCase {
 	private static final Logger LOGGER = Logger.getLogger(BlockScorerITCase.class.getName());
 
-	private static final byte[] HASH_BYTES = new byte[] {
-			(byte)0xF7, (byte)0xF6, (byte)0xF5, (byte)0xF4, (byte)0xF3, (byte)0xF2, (byte)0xF1, (byte)0xF0,
-			(byte)0xE7, (byte)0xE6, (byte)0xE5, (byte)0xE4, (byte)0xE3, (byte)0xE2, (byte)0xE1, (byte)0xE0,
-			(byte)0xD7, (byte)0xD6, (byte)0xD5, (byte)0xD4, (byte)0xD3, (byte)0xD2, (byte)0xD1, (byte)0xD0,
-			(byte)0xC7, (byte)0xC6, (byte)0xC5, (byte)0xC4, (byte)0xC3, (byte)0xC2, (byte)0xC1, (byte)0xC0
+	private static final byte[] HASH_BYTES = new byte[]{
+			(byte) 0xF7, (byte) 0xF6, (byte) 0xF5, (byte) 0xF4, (byte) 0xF3, (byte) 0xF2, (byte) 0xF1, (byte) 0xF0, // preserve-newline
```
[the remark this answers is not in the document]
Comment (Jaguar0625) [reply nis/src/it/java/org/nem/nis/BlockScorerITCase.java:22]: no that is entered manually, to preserve readability. 
without that, it groups 9 byte rows, which is ... unreadable.

File: core/pom.xml [hunk 2]
```diff
@@ -267,29 +267,34 @@
...
+                        </includes>
+
+                        <importOrder />
+                        <removeUnusedImports />
+                        <trimTrailingWhitespace/>
+                        <endWithNewline/>
+
                         <eclipse>
-                          <version>4.9.0</version>
+                          <version>4.13</version>
```
[the remark this answers is not in the document]
Comment (Jaguar0625) [reply core/pom.xml:289]: some of the other poms had 4.13 already, so i used that one everywhere and it seemed to work

File: nis/src/it/java/org/nem/nis/pox/poi/PoiImportanceCalculatorITCase.java [hunk 3]
```diff
@@ -415,12 +418,12 @@ public void poiCalculationIsFastEnough() {
 
 		// Assert
 		// > context setup needs 646ms
-		//   - AccountProcessor ctor needs 127ms
-		//   - setup vested balance vector needs 24ms
-		//   - add reverse links to allow net calculation needed 95ms
-		//   - update the matrix with all net outflows needed 281ms
-		//   - removeLessThan and normalizeColumns needed 12ms
-		//   - buildInterLevelProximityMatrix needed 107ms
+		// - - AccountProcessor ctor needs 127ms
```
[the remark this answers is not in the document]
Comment (Jaguar0625) [reply nis/src/it/java/org/nem/nis/pox/poi/PoiImportanceCalculatorITCase.java:421]: that is intentional to force the indentation. 
spotless removes leading indentation from comments, so i think the extra indent (with two -) is a bit more readable than only one - (unindented).

**Pull request merged**